src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        action="/"
+        class="form"
+        data-qa="big"
+        method="get"
+        >
+        <label
+          for="big-search"
+          class="image"
+        >
+          <img
+            src="images/search.svg"
+            alt="search"
+          >
+        </label>
+        <input
+          class="search style"
+          id="big-search"
+          type="search"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
Consider adding a `label` element for the search input for better accessibility. 🔍